README.rdoc

```diff
@@ -46,6 +46,19 @@ circle.yml:
             files:
               - test/**/*.rb
 
+Alternatively, you can add the following code to your test runner setup:
+
+    # test/helper.rb
+    require 'minitest'
+    require 'minitest/autorun'
+    require 'minitest/ci'
+
+    Minitest::Ci.report_dir = "#{ENV["CIRCLE_TEST_REPORTS"]}/reports"
```

Comment:
It appears both [us][us] and [Travis][travis] set a `CI` environment variable.

[us]: https://circleci.com/docs/environment-variables/#basics
[travis]: https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables